docs/docs/personalization.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Personalization
+---
+
+## What is Personalization?
+
+- Have you ever landed on a website that greeted you by name?
+
+- Did you ever search for a book and realize the suggested results were based on your previous purchases?
+
+These are examples of personalization. Personalization adjusts the content of a website, and creates a custom experience for users. With personalization, one website can offer a variety of experiences to different visitors.
+
+## Why is Personalization Important?
+
+If you are [creating an e-commerce site](/tutorial/ecommerce-tutorial/), personalization is key in converting your visitors into buyers. Online merchants use personalization as a way to suggest relevant items to a shopper.
+
+Outside of e-commerce, personalization is important for other types of websites, like e-learning. Personalization, if implemented strategically, can help educational sites deliver more targeted learning experiences. For example, if a learner is struggling with learning the basics of JavaScript, the website can give the learner additional JavaScript practice problems.
+
+## Using Personalization with Gatsby
+
+Personalization contains dynamic content. Personalization is possible in Gatsby because Gatsby can enhance static content by [React Hydration](/blog/2018-10-15-beyond-static-intro/#hydration). During build time Gatsby invokes the React server-side DOM to generate content.
```

Comment:
This is a great start, since personalization is such an under-documented/supported area of Gatsby. Here's a suggestion on how to restructure it to emphasize what Gatsby _can_ do while being realistic about the situation:
```suggestion
Personalization involves rendering dynamic content to a user. The way Gatsby generates static content at [build](/docs/glossary#build) time and [hydrates into a React web application](/blog/2018-10-15-beyond-static-intro/#hydration) both limits what can be done to render dynamic content at build time, and creates opportunities to enhance content with client-side JavaScript.

One possible way to go about personalization is through lazy-loading of content with client-side JavaScript, by requesting specific content from a web/cloud service or [lambda function](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html) and displaying it to a user when they view a page. A benefit of this approach would be the ability to access dynamic content per user with your Gatsby site, with minimal infrastructure changes. A drawback would be that it relies on client-side JavaScript to render content, which could change from the initial server-rendered view and cause a flash of changed content.
```

When we have a specific doc on hydration, we could cover this use case. cc @gillkyle.

Reply:
Should wenn not only show drawbacks but also mention the benefits: if a wireframe is loaded fast from prebuild files then the user get an fast feedback. 

Facebook make it also this way and they show a placeholder while loading user content